<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MenuPartida</class>
 <widget class="QWidget" name="MenuPartida">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background: qlineargradient(x1:0, y1:0, x2:1, y2:1, stop:0 #2c3e50, stop:1 #3498db);</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="1" column="1">
    <widget class="QFrame" name="frameConfiguracion">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="styleSheet">
      <string notr="true">background: rgba(255,255,255,0.15);
border: 2px solid rgba(255,255,255,0.3);
border-radius: 15px;</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::Shape::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Shadow::Raised</enum>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QFrame" name="frameJugadores">
        <property name="sizePolicy">
         <sizepolicy hsizetype="MinimumExpanding" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="styleSheet">
         <string notr="true">background: rgba(255,255,255,0.1);
border: 1px solid rgba(255,255,255,0.2);
border-radius: 10px;</string>
        </property>
        <property name="frameShape">
         <enum>QFrame::Shape::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Shadow::Raised</enum>
        </property>
        <layout class="QGridLayout" name="gridLayout_3">
         <item row="0" column="0">
          <widget class="QLabel" name="lblJugadores">
           <property name="styleSheet">
            <string notr="true">color: white; font-size: 16px; font-weight: bold;</string>
           </property>
           <property name="text">
            <string>NÃšMERO DE JUGADORES:</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QComboBox" name="comboJugadores">
           <property name="maximumSize">
            <size>
             <width>55</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">QComboBox {
    background: white;
	color: black;
    border: 2px solid #bdc3c7;
    border-radius: 5px;
    padding: 5px;
    font-size: 16px;
    min-width: 100px;
}
QComboBox::drop-down {
    subcontrol-origin: padding;
    subcontrol-position: top right;
    width: 20px;
    border-left-width: 1px;
    border-left-color: #bdc3c7;
    border-left-style: solid;
    border-top-right-radius: 5px;
    border-bottom-right-radius: 5px;
}
QComboBox::down-arrow {
    image: url(down_arrow.png);
}
QComboBox QAbstractItemView {
    background: white;
	color: black;
    border: 1px solid #bdc3c7;
    selection-background-color: #3498db;
}</string>
           </property>
           <item>
            <property name="text">
             <string>2</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>3</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>4</string>
            </property>
           </item>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QLineEdit" name="editJugador1">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>100</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">QLineEdit {
    background: white;
	color: black;
    border: 2px solid #bdc3c7;
    border-radius: 5px;
    padding: 5px;
    font-size: 14px;
    min-width: 200px;
}
QLineEdit:focus {
    border-color: #3498db;
}</string>
           </property>
           <property name="text">
            <string>Jugador 1</string>
           </property>
           <property name="placeholderText">
            <string>Jugador 1</string>
           </property>
          </widget>
         </item>
         <item row="6" column="1">
          <widget class="QLineEdit" name="editJugador3">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>100</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">QLineEdit {
    background: white;
	color: black;
    border: 2px solid #bdc3c7;
    border-radius: 5px;
    padding: 5px;
    font-size: 14px;
    min-width: 200px;
}
QLineEdit:focus {
    border-color: #3498db;
}</string>
           </property>
           <property name="text">
            <string>Jugador 3</string>
           </property>
           <property name="placeholderText">
            <string>Jugador 3</string>
           </property>
          </widget>
         </item>
         <item row="8" column="1">
          <widget class="QLineEdit" name="editJugador4">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>100</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">QLineEdit {
    background: white;
	color: black;
    border: 2px solid #bdc3c7;
    border-radius: 5px;
    padding: 5px;
    font-size: 14px;
    min-width: 200px;
}
QLineEdit:focus {
    border-color: #3498db;
}</string>
           </property>
           <property name="text">
            <string>Jugador 4</string>
           </property>
           <property name="placeholderText">
            <string>Jugador 4</string>
           </property>
          </widget>
         </item>
         <item row="2" column="1">
          <widget class="QLineEdit" name="editJugador2">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>100</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">QLineEdit {
    background: white;
	color: black;
    border: 2px solid #bdc3c7;
    border-radius: 5px;
    padding: 5px;
    font-size: 14px;
    min-width: 200px;
}
QLineEdit:focus {
    border-color: #3498db;
}</string>
           </property>
           <property name="text">
            <string>Jugador 2</string>
           </property>
           <property name="placeholderText">
            <string>Jugador 2</string>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="lblJugador1">
           <property name="minimumSize">
            <size>
             <width>120</width>
             <height>0</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>100</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">color: white; font-size: 14px;</string>
           </property>
           <property name="text">
            <string>JUGADOR 1:</string>
           </property>
          </widget>
         </item>
         <item row="2" column="0">
          <widget class="QLabel" name="lblJugador2">
           <property name="minimumSize">
            <size>
             <width>120</width>
             <height>0</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>100</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">color: white; font-size: 14px;</string>
           </property>
           <property name="text">
            <string>JUGADOR 2:</string>
           </property>
          </widget>
         </item>
         <item row="6" column="0">
          <widget class="QLabel" name="lblJugador3">
           <property name="minimumSize">
            <size>
             <width>120</width>
             <height>0</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>100</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">color: white; font-size: 14px;</string>
           </property>
           <property name="text">
            <string>JUGADOR 3:</string>
           </property>
          </widget>
         </item>
         <item row="8" column="0">
          <widget class="QLabel" name="lblJugador4">
           <property name="minimumSize">
            <size>
             <width>120</width>
             <height>0</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>100</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">color: white; font-size: 14px;</string>
           </property>
           <property name="text">
            <string>JUGADOR 4:</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QFrame" name="frameConfigJuego">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="styleSheet">
         <string notr="true">background: rgba(255,255,255,0.1);
border: 1px solid rgba(255,255,255,0.2);
border-radius: 10px;</string>
        </property>
        <property name="frameShape">
         <enum>QFrame::Shape::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Shadow::Raised</enum>
        </property>
        <layout class="QGridLayout" name="gridLayout_2">
         <item row="1" column="1">
          <widget class="QSpinBox" name="spinTamTablero">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="maximumSize">
            <size>
             <width>55</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">QSpinBox {
    background: white;
	color: black;
    border: 2px solid #bdc3c7;
    border-radius: 5px;
    padding: 5px;
    font-size: 16px;
    min-width: 60px;
}
QSpinBox:disabled {
	background-color: #f0f0f0;
	color: #888888;
	border: 2px solid #cccccc;
}
QSpinBox:enabled {
	background-color: white;
	color: black;
	border: 2px solid #3498db;
}</string>
           </property>
           <property name="minimum">
            <number>63</number>
           </property>
           <property name="maximum">
            <number>90</number>
           </property>
          </widget>
         </item>
         <item row="0" column="0">
          <widget class="QCheckBox" name="checkPersonalizado">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize">
            <size>
             <width>215</width>
             <height>0</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>100</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">QCheckBox {
    color: white;
    font-size: 14px;
    font-weight: bold;
    spacing: 8px;
}
QCheckBox::indicator {
    width: 20px;
    height: 20px;
}
QCheckBox::indicator:unchecked {
    background: white;
    border: 2px solid #bdc3c7;
    border-radius: 3px;
}
QCheckBox::indicator:checked {
    background: qlineargradient(x1:0, y1:0, x2:0, y2:1, stop:0 #9b59b6, stop:1 #8e44ad);
    border: 2px solid #8e44ad;
    border-radius: 3px;
}</string>
           </property>
           <property name="text">
            <string>TABLERO PERSONALIZADO</string>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="lblTamTablero">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize">
            <size>
             <width>215</width>
             <height>0</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>100</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">color: white; font-size: 14px; font-weight: bold;</string>
           </property>
           <property name="text">
            <string>TAMAÃ‘O DEL TABLERO</string>
           </property>
          </widget>
         </item>
         <item row="2" column="0">
          <widget class="QCheckBox" name="checkDosDados">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize">
            <size>
             <width>215</width>
             <height>0</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>100</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">QCheckBox {
    color: white;
    font-size: 14px;
    font-weight: bold;
    spacing: 8px;
}
QCheckBox::indicator {
    width: 20px;
    height: 20px;
}
QCheckBox::indicator:unchecked {
    background: white;
    border: 2px solid #bdc3c7;
    border-radius: 3px;
}
QCheckBox::indicator:checked {
    background: qlineargradient(x1:0, y1:0, x2:0, y2:1, stop:0 #9b59b6, stop:1 #8e44ad);
    border: 2px solid #8e44ad;
    border-radius: 3px;
}</string>
           </property>
           <property name="text">
            <string>USAR DOS DADOS</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QFrame" name="frameTitulo">
     <property name="styleSheet">
      <string notr="true">background: qlineargradient(x1:0, y1:0, x2:1, y2:0, stop:0 #1abc9c, stop:1 #16a085);
border: 3px solid #16a085;
border-radius: 20px;</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::Shape::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Shadow::Raised</enum>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QLabel" name="lblTitulo">
        <property name="styleSheet">
         <string notr="true">color: white;
font-family: 'Arial Rounded MT Bold', 'Arial', sans-serif;
font-size: 32px;
font-weight: bold;
text-shadow: 2px 2px 4px rgba(0,0,0,0.5);</string>
        </property>
        <property name="text">
         <string>PREPARACIÃ“N DE PARTIDA</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignmentFlag::AlignCenter</set>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="0">
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Orientation::Horizontal</enum>
     </property>
     <property name="sizeType">
      <enum>QSizePolicy::Policy::Fixed</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>100</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="1" column="2">
    <spacer name="horizontalSpacer_2">
     <property name="orientation">
      <enum>Qt::Orientation::Horizontal</enum>
     </property>
     <property name="sizeType">
      <enum>QSizePolicy::Policy::Fixed</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>100</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="2" column="0" colspan="3">
    <widget class="QFrame" name="frameBotones">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="styleSheet">
      <string notr="true">background: transparent; border: none;</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::Shape::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Shadow::Raised</enum>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QPushButton" name="btnComenzarPartida">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="styleSheet">
         <string notr="true">#btnComenzarPartida {
    background: qlineargradient(x1:0, y1:0, x2:0, y2:1, stop:0 #27ae60, stop:1 #2ecc71);
    border: 2px solid #219653;
    border-radius: 10px;
    color: white;
    font-family: 'Arial';
    font-size: 16px;
    font-weight: bold;
    padding: 12px 25px;
    min-width: 120px;
}
#btnComenzarPartida:hover {
    background: qlineargradient(x1:0, y1:0, x2:0, y2:1, stop:0 #229954, stop:1 #27ae60);
}</string>
        </property>
        <property name="text">
         <string>COMENZAR</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btnCancelarPartida">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="styleSheet">
         <string notr="true">#btnCancelarPartida {
    background: qlineargradient(x1:0, y1:0, x2:0, y2:1, stop:0 #e67e22, stop:1 #d35400);
    border: 2px solid #e67e22;
    border-radius: 10px;
    color: white;
    font-family: 'Arial';
    font-size: 16px;
    font-weight: bold;
    padding: 12px 25px;
    min-width: 120px;
}
#btnCancelarPartida:hover {
    background: qlineargradient(x1:0, y1:0, x2:0, y2:1, stop:0 #d35400, stop:1 #e67e22);
}</string>
        </property>
        <property name="text">
         <string>CANCELAR</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
